packages/eslint-config-godaddy-minimal/index.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+  extends: [
+    require('./extends')('eslint-config-godaddy-react-flow')
+  ],
+  rules: {
+    'complexity': 0,
+    'consistent-return': 0,
+    'id-length': 0,
+    'indent': [2, 2, {
+      VariableDeclarator: { var: 2, let: 2, const: 3 },
+      flatTernaryExpressions: true,
+      MemberExpression: 'off'
+    }],
+    'max-params': 0,
+    'max-nested-callbacks': 0,
+    'max-statements': 0,
+    'no-else-return': 0,
+    'no-nested-ternary': 0,
```

Comment:
Want to add a +1 for disabling this rule, one of my biggest pet peeves.  Ternaries can be used to make code a lot more readable compared to cluttering if/then/else chains or switch statements.  Having to constantly disable this rule is a pain.